packages/core/src/config.ts

```diff
@@ -930,6 +930,7 @@ const composeDtsConfig = async (
         // Only setting ⁠dts.bundle to true will generate the bundled d.ts.
         bundle: dts?.bundle ?? false,
         distPath: dts?.distPath ?? output?.distPath?.root ?? './dist',
+        build: dts?.build ?? false,
```

Comment:
Sound not very good since the js output is not emitted by tsc, so the output of dependencies of the project only contains DTS. This option should be enabled by user manualy to align with the Modern.js Module. The former version of Modern.js Module has enabled it by default when composite is true, but it causes a lot of isses so in later version, a configuration is added